### DISCUSSION TITLE
Developer documentation

### DISCUSSION THREAD

**Discussion by dmdunla** [sandialabs/pyttb#57] [Ideas]
- How should we document the development process?
- Are there good examples of developer documentation that we can emulate?
- How do we document the use opiniated dev/testing tools like those adapted as part of PR #54 ?

**Comment by ntjohnson1** [comment]
I think there are tons of projects with various constraints that lead to how they share dev docs. I think pydantics is pretty nice https://docs.pydantic.dev/contributing/ they seem to be entirely make driven but still something to emulate to reduce the barrier to entry as much as possible. My guess is one of the steps for the developer process is to have people sign the waiver of liability details. On the read the docs it says to contact you if things can't be resolved on github, should that be how we propose that step for the dev docs as well?

**Comment by ntjohnson1** [comment]
Are there more things we are exploring here or can we close the discussion? I think our dev docs are updated and there are tickets for tutorials/user docs
